study-builder/studies/testboston/snippets/consent-section4.conf

```diff
@@ -1,336 +0,0 @@
-{
```

Comment:
We now have just 3 sections, so this file is basically renamed to `consent-section3.conf`.